nss-day-cohort-46/reactive-nutshell-gauntlets-of-adventure-1
# Nah edit tasks

# Description

- dataProvider has been updated.
- passing taskId in TaskForm
- edit feature is working / need to work on populating editForm with current data.
- delete button is working
- complete button marks item as completed / need to remove from DOM when 'completed'
- tasks only show up for the user that is logged in

Fixes # (issue)
Edit button


Please delete options that are not relevant.


# Testing Instructions

git fetch --all
git checkout nah-editTasks
serve: npm start
API: make sure API is running

once on my branch.. you can create a new task, edit a task, delete a task and mark it as completed.

# Checklist:
